packages/suite/src/views/wallet/coinmarket/index.tsx

```diff
@@ -254,6 +254,7 @@ export const CoinmarketAmountWrapperText = styled.div`
     font-variant-numeric: tabular-nums;
     overflow: hidden;
     text-overflow: ellipsis;
+    line-height: 1;
```

Comment:
Couldn't we do it without explicitly setting line-height? 🙏

Reply:
There is a problem with Icon centering and this solution helps. Strange bug  🤷‍♂️